.github/README.md

```diff
@@ -1,26 +1,20 @@
 <h1 align="center">NvChad</h1>
 
 <div align="center">
-	<a href="https://nvchad.github.io/">Home</a>
+	<a href="https://nvchad.com/">Home</a>
   <span> • </span>
-    	<a href="https://nvchad.github.io/quickstart/install">Install</a>
+    	<a href="https://nvchad.com/#/docs/quickstart/install">Install</a>
   <span> • </span>
-       	<a href="https://nvchad.github.io/contribute">Contribute</a>
+       	<a href="https://nvchad.com/#/docs/contribute">Contribute</a>
   <span> • </span>
 	<a href="https://github.com/NvChad/NvChad#gift_heart-support">Support</a>
   <span> • </span>
-        <a href="https://nvchad.github.io/Features">Features</a>
+        <a href="https://nvchad.com/#/docs/features">Features</a>
   <p></p>
 </div> 
 
 <div align="center">
  
-[![Super Linter](https://img.shields.io/github/workflow/status/NvChad/NvChad/Super-Linter/main?style=flat-square&logo=github&label=Build&color=8DBBE9)]()
-<a href="https://github.com/NvChad/NvChad/blob/main/LICENSE"
-        ><img
-            src="https://img.shields.io/github/license/NvChad/NvChad?style=flat-square&logo=GNU&label=License&color=df967f"
-            alt="License"
-    />
 [![Neovim Minimum Version](https://img.shields.io/badge/Neovim-0.8.0-blueviolet.svg?style=flat-square&logo=Neovim&color=90E59A&logoColor=white)](https://github.com/neovim/neovim)
```

Comment:
Docs say minimum version is `0.8.3`. Which one should stay?

Reply:
@mikededo need to update the readme, thx for correcting!

Reply:
I was looking through the docs today and I found a couple of improvements, do you want me to later create a PR with it? I would be willing to help with the docs 💪

Reply:
@mikededo feel free to!  Btw do know that all the docs are in md files, `/docs and /news` format.